_config.yml

```diff
@@ -147,7 +152,12 @@ t:
         title: "Atraksis"
         url: "https://atraksis.fr/"
         logo: "https://atraksis.fr/wp-content/themes/atraksis/assets/img/logo-atraksis-footer.svg"
-        content: "Our partner for field testing phases, and our intermediary with firefighters."
+        content: "A stakeholder for rescue innovation, and our point of contact with firefighters."
+      sponsor3:
+        title: "SDIS Ardèche"
+        url: "https://www.sdis07.fr/"
+        logo: "https://www.sdis07.fr/ardeche-secours-incendie/themes/sapeurs-pompiers-ardeche/medias/img/logo-sdis07-pompiers-ardeche.png"
+        content: "Our partner for the collection of needs and field experimentation."
```

Comment:
"collection of needs" unfortunately doesn't translate directly from French :sweat_smile: 

in engineering, usually, it's "requirements elicitation"

Reply:
Yes, collection of need is definitely not ok 😅
"Requirements gathering" sounds more common to me ! As you want !

Reply:
yes, go for it!